tests/unit/lms/views/api/blackboard/files_test.py

```diff
@@ -174,12 +171,12 @@ def test_it_when_file_not_in_course(
         self, view, blackboard_api_client, course_service, course_copy_plugin
     ):
         course_copy_plugin.is_file_in_course.return_value = False
-        blackboard_api_client.public_url.side_effect = BlackboardFileNotFoundInCourse(
-            file_id="FILE_ID"
+        blackboard_api_client.public_url.side_effect = FileNotFoundInCourse(
+            "blackboard_file_not_found_in_course", file_id="FILE_ID"
         )
```

Comment:
Something is up here. This test still passes if you remove this line?

I was investigating having to instantiate the error correctly, but it turns out it doesn't matter if you set it at all in this test?